### ISSUE TITLE
Refactor 1.x content (or remove it)

### ISSUE THREAD

**Issue by guardrex** [aspnet/Docs#9270]
While working on #9268, I noticed some problems with the 1.x versioning.

The versioning that appears for 1.x points to a 2.0 sample, so it's incorrect in the topic. See this 1.1 version ...

https://docs.microsoft.com/aspnet/core/tutorials/getting-started-with-swashbuckle?view=aspnetcore-1.1&tabs=visual-studio%2Cvisual-studio-xml#xml-comments

... scroll down a bit, and you'll see a `CreateDefaultBuilder` in there.

It also looks like the 1.x links to the sample provide a 2.0 project file.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 14fce65f-1337-58cb-2193-8d8e0e62792d
* Version Independent ID: 59222460-a971-e76e-4fe9-23d4d7d768ee
* Content: [Get started with Swashbuckle and ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/tutorials/getting-started-with-swashbuckle?view=aspnetcore-2.1&tabs=visual-studio%2Cvisual-studio-xml)
* Content Source: [aspnetcore/tutorials/getting-started-with-swashbuckle.md](https://github.com/aspnet/Docs/blob/master/aspnetcore/tutorials/getting-started-with-swashbuckle.md)
* Product: **aspnet-core**
* GitHub Login: @zuckerthoben
* Microsoft Alias: **scaddie**

**Comment by scottaddie** [comment]
@guardrex You're correct that the *Program.cs* sample displayed for 1.1 is incorrect.

**Comment by guardrex** [comment]
I would've fixed it on the spot, but I think the project file snippet situation needs work, too. Best to work it separately from the other issue, which has priority bug fixes that should go in quickly-ish.

**Comment by Rick-Anderson** [comment]
Can't we remove the 1.x content?

**Comment by guardrex** [comment]
Earlier, we said we'd go with PDFs; but as we get closer to mid-year (June 27 2019 ... EOS for 1.x), I guess we'll be killing it more and more without going to the trouble of PDFing the 1.x bits.
